postman/manifest.json

```diff
@@ -0,0 +1,34 @@
+{
+  "display_name": "Postman",
+  "maintainer": "integrations-partnerships@postman.com",
+  "manifest_version": "1.0.0",
+  "name": "postman",
+  "metric_prefix": "PostmanMonitor_",
+  "metric_to_check": "",
```

Comment:
This should be populated with a metric expected to always be reported to determine if the check is working correctly.

Reply:
Updated. Also updated the metric_prefix according to the new naming convention.